components/api_server/src/routes/health.py

```diff
@@ -0,0 +1,9 @@
+"""Health end-point."""
+
+import bottle
+
+
+@bottle.get("/api/internal/health", authentication_required=False)
+def get_health():
+    """Return the API-server health."""
+    return {"healthy": True}
```

Comment:
Perhaps useful to inline comment that we don't monitor any additional processes, so up=up

Reply:
Done.